src/components/Alert/InlineAlert.test.tsx

```diff
@@ -7,20 +7,20 @@ describe('Inline Alerts', () => {
         const cut = mount(<InlineAlert variant="info">🤔</InlineAlert>);
         expect(cut.find('.rvt-inline-alert').hasClass("rvt-inline-alert--info")).toBe(true);
     });
-    it('should have "valid" styling with the valid variant', () => { 
-        const cut = mount(<InlineAlert variant="valid">😎</InlineAlert>);
+    it('should have "success" styling with the valid variant', () => { 
```

Comment:
```suggestion
    it('should have "success" styling with the success variant', () => { 
```
The second part of the description on these tests should probably also say "Success."